src/libhirte/bus/utils.c

```diff
@@ -80,11 +80,31 @@ UnitInfo *new_unit() {
         }
 
         unit->ref_count = 1;
+        unit->is_from_bus_msg = false;
         LIST_INIT(units, unit);
 
         return steal_pointer(&unit);
 }
 
+int copy_unit(UNUSED UnitInfo *dest, UNUSED UnitInfo *src) {
+        int r = 0;
+        if (src->node != NULL) {
+                asprintf(&dest->node, "%s", src->node);
+        }
+        asprintf(&dest->id, "%s", src->id);
```

Comment:
The RPM build is throwing errors as the result from `asprintf` is not used. I think you can just use `strdup`here - at least its used in other code files when just copying values.

Reply:
Yeah, I've seen that. Interesting that it does not happen when building locally. Anyway, it will be changed